<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arbin data processor v0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo_alpha.png</normaloff>logo_alpha.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="files_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select files...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_textpaths">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Use filepaths contained within 'cell_info.csv'. Files other than 'cell_info.csv' are ignored from initial folder or file selection. </string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Use 'cell_info.csv' filepaths</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>80</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_sv_avg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>203</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Save average data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_sv_step">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>203</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Save capacity 'per step' in addition to the default 'per cycle'.</string>
     </property>
     <property name="text">
      <string>Save step data</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_sv_indv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>203</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Saves a single file for each converted capacity type, i.e. mAh/g, mAh/cm2 and mAh/cm3.</string>
     </property>
     <property name="text">
      <string>Save conversion data individually</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>150</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_filenames">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>203</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Use original filename instead of the name provided in 'cell_info.csv' when exporting files.</string>
     </property>
     <property name="text">
      <string>Use filename as cell name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_calc_avg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>203</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Calculates average data from data files.</string>
     </property>
     <property name="text">
      <string>Calculate average data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="usr_avg_name">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>120</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>average_data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Used to name exported files containing the average data.</string>
     </property>
     <property name="text">
      <string>Average name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="run_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="folder_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select folder...</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>30</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
